CHANGELOG.md

```diff
@@ -28,6 +28,10 @@
 * Admin bar menu items can now specify a custom Vue component to be used in place of `AposButton`.
 * Sets `username` fields to follow the user `title` field to remove an extra step in user creation.
 * Adds default data to the `outerLayoutBase.html` `<title>` tag: `data.piece.title or data.page.title`.
+* Moves the core UI build task into the start up process. The UI build runs when:
+  1. The build folder does not yet exist.
+  2. The package.json file is newer than the existing UI build.
```

Comment:
package-lock.json would make more sense.

Reply:
I thought about that, but if they use yarn it there wouldn't be one. We could look for yarn.lock as well, but then it feels like we're chasing technologies.

Though you're right that a lock file update could indicate a need to rebuild the UI....

Reply:
Interesting point about chasing technologies. You could investigate whether all updates bump the semver ranges in package.json with a new minimum version, but I suspect they do not. So we might have to chase these two. I don't think a third major npm client is likely.